### PR TITLE
[VAS-1195] New CI IBANs Mongo Collection

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

➕ Added a new collection `creditorInstitutionIbans` 
✏️ Edited the cron job to save the ibans in the new collection

Extra
🛡️ **CODEOWNERS** file edited: added `pagopa-core-team`

#### Motivation and Context

[Jira Ticket ](https://pagopa.atlassian.net/browse/VAS-1195)

#### How Has This Been Tested?

✋ Manually

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: openapi/openapi.json
Comment (gioelemella) [inline openapi/openapi.json:1]: Format the openapi

**Review by alessio-cialini — APPROVED** [review]

**Review by gioelemella — CHANGES_REQUESTED** [review]
Format the openapi

File: openapi/openapi.json
Comment (jacopocarlini) [reply openapi/openapi.json:1]: done

**Review by gioelemella — APPROVED** [review]
💯

**Pull request merged**